components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -95,19 +97,22 @@ public String jwks() {
             while (enumeration.hasMoreElements()) {
                 String alias = (String) enumeration.nextElement();
                 if (keystore.isKeyEntry(alias)) {
-                    Certificate cert = keystore.getCertificate(alias);
-                    certificatesWithAliases.put(alias, cert);
+                    CertificateInfo certificateInfo = new CertificateInfo();
```

Comment:
set alias and certificate within the constructor, they are mandatory for a certificate